bin/run-tests-in-docker.sh

```diff
@@ -16,7 +16,7 @@
 set -e
 
 # Build the Docker image
-docker build --rm -t exercism/test-runner .
+docker build --rm -t exercism/idris-test-runner .
```

Comment:
https://github.com/exercism/lfe-test-runner/blob/017f86a6c1d8c844a3531e8bd0613c10b537d8f5/bin/run-in-docker.sh#L34 has `docker build --rm -t exercism/test-runner .` so I think you need to revert this edit.

Reply:
Fixed!